Sources/Build/PackageVersion.swift

```diff
@@ -33,13 +33,17 @@ func saveRootPackage(_ dirPath: String, package: Package) throws {
     let versionSha = try repo.versionSha(tag: "\(prefix)\(version)")
 
     var data = packageVersionData(package)
+    data += "public let sha: String?"
     if headSha != versionSha {
-        data += "public let sha: String = \"\(headSha)\" \n"
+        data += " = \"\(headSha)\""
     }
+    data += "\n"
+    data += "public let modified: String?"
     if repo.hasLocalChanges {
-        //TODO: save time
-        data += "public let modified: String = \"\" \n"
+        let date = try strftime("%Y-%m-%dT%XZ", time: gmtime_r())
+        data += " = \"\(date)\""
     }
+    data += "\n"
```

Comment:
Optionals are nil by default. I thought `public let modified: String?` would be enough.